candidates/views/other_names.py

```diff
@@ -63,8 +64,34 @@ def form_valid(self, form):
             )
             self.person.extra.record_version(change_metadata)
             self.person.extra.save()
+            if self.request.is_ajax():
```

Comment:
This block puzzled me a bit on first reading, so a comment explaining why it's here at the top would have been helpful to me. (i.e. that it's useful when this is an AJAX usage to return all the alternative names for that person, if I've got that right.